USING.md

```diff
@@ -25,14 +24,21 @@ The following modules are applications and data for testing and working with JTS
 To include JTS in a Maven project, add a dependency block like the following:
 
 ```xml
+<properties>
+    <jts.version>1.16.1</jts.version>
+</properties>
 <dependency>
     <groupId>org.locationtech.jts</groupId>
     <artifactId>jts-core</artifactId>
     <version>${jts.version}</version>
 </dependency>
 ```
 
-JTS snapshot artifacts are published to the LocationTech Maven repository. To include JTS in a project, add the following repositories to the pom.
+JTS artifacts are available on maven central.
+
+### Using JTS Snapshots
+
+Our [build server](https://ci.eclipse.org/jts/) publishes to the LocationTech Maven repository. To include JTS in a project, add the following repositories to the pom:
```

Comment:
Maybe change link to JTS artifacts:  https://ci.eclipse.org/jts/job/JTS_deploy_snapshots/